gnmvidispine/vs_item.py

```diff
@@ -113,7 +114,7 @@ def createPlaceholder(self,metadata=None,group=None):
                                     query={'container': 1, 'video': 1,
                                     })
         except VSBadRequest as e:
-            print rqbody
+            print(rqbody)
```

Comment:
needs to be changed to something like `logger.warning("Got a bad request for {0}".format(rqbody))`

Reply:
Done.